dist/windows/README.md

```diff
@@ -1,6 +1,6 @@
 # radare2-installer
 Windows installer scripts for radare2
 
-[![Build Status](http://ci.rada.re/buildStatus/icon?job=radare2-win-installer)](http://ci.rada.re/job/radare2-win-installer/)
+[![Build Status](https://ci.rada.re/buildStatus/icon?job=radare2-win-installer)](https://ci.rada.re/job/radare2-win-installer/)
 
-[Radare.exe](http://bin.rada.re/radare2.exe)
+[Radare.exe](https://bin.rada.re/radare2.exe)
```

Comment:
that url is invalid, and the bin.rada.re server is deprecated, ill update that readme to match the new reality